handlers/LunchHandler.js

```diff
@@ -121,6 +140,89 @@ export default class LunchHandler extends BaseHandler {
 		);
 	}
 
+	getHotPotMenu() {
```

Comment:
The three getHotPotMenu, getPahadPoisidMenu, getUTMenu code is 99% the same. Create a method similar to getFacebookMenu that takes the changing parts as arguments.